app/coffee/DocManager.coffee

```diff
@@ -5,24 +5,19 @@ _ = require "underscore"
 async = require "async"
 
 module.exports = DocManager =
-	getDoc: (project_id, doc_id, options, callback = (error, doc, mongoPath) ->) ->
-		if typeof(options) == "function"
-			callback = options
-			options.include_deleted = false
+	getDoc: (project_id, doc_id, callback = (error, doc, mongoPath) ->) ->
```

Comment:
This method will now always return deleted docs if they exist. Is that correct? If so, can you explain why this is not a problem?